### PR TITLE
provider/aws: Prevent crashing when deleting ecs_service which is gone

### PR DESCRIPTION
**Pull request by radeksimko**
fixes https://github.com/hashicorp/terraform/issues/3868 and also addresses https://github.com/hashicorp/terraform/pull/3829#issuecomment-155350232

### CODE REVIEW

**Comment by stack72** [comment]
Looks good @radeksimko :+1:

File: builtin/providers/aws/resource_aws_ecs_service.go [hunk 1]
```diff
@@ -156,6 +156,8 @@ func resourceAwsEcsServiceRead(d *schema.ResourceData, meta interface{}) error {
 	}
 
 	if len(out.Services) < 1 {
+		log.Printf("[DEBUG] Removing ECS service %q because it's gone", service.ServiceArn)
```
Comment (catsby) [inline builtin/providers/aws/resource_aws_ecs_service.go]: `service` here is undefined :dizzy_face: 

```
1 errors occurred:
--> darwin/amd64 error: exit status 2
Stderr: # github.com/hashicorp/terraform/builtin/providers/aws
builtin/providers/aws/resource_aws_ecs_service.go:159: undefined: service
```

Comment (catsby) [inline builtin/providers/aws/resource_aws_ecs_service.go]: I'm fixing this...

Comment (radeksimko) [inline builtin/providers/aws/resource_aws_ecs_service.go]: doh... sorry about that!

Comment (radeksimko) [inline builtin/providers/aws/resource_aws_ecs_service.go]: I wonder why Travis didn't pick this up btw.

Comment (catsby) [inline builtin/providers/aws/resource_aws_ecs_service.go]: Fixed in https://github.com/hashicorp/terraform/pull/3943

Comment (radeksimko) [inline builtin/providers/aws/resource_aws_ecs_service.go]: Thx :+1:

Comment (catsby) [inline builtin/providers/aws/resource_aws_ecs_service.go]: > I wonder why Travis didn't pick this up btw.

I was wondering the same thing. Very strange :confused: Oh well, it's fixed, thanks for the follow up PR :smile:

**Pull request merged**